### PR TITLE
MILESTONE: PORFOLIO MOBILE MENU

### PR DESCRIPTION
**Pull request by Amedzro-Elikplim**
# MOBILE MENU

- Add mobile menu to mobile view
- Style mobile menu using a flexbox layout
- Create a separate CSS file for the mobile menu
- Add vanilla Js code to toggle mobile menu visible and hidden
- Hide the mobile menu on nav links click
- Style nav bar with position fixed 
- Select all li in ul using querySelectorAll
- Add functionality that scrolls to specified section when a nav link is clicked

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
       name="description"
       content="Author: Amedzro Emmanuel Elikplim, School: Microverse, Project: Portfolio"
     />
-
+    <link rel="stylesheet" href="menu.css" />
     <link rel="stylesheet" href="styles.css" />
 
     <link rel="preconnect" href="https://fonts.googleapis.com" />
```
Comment (Meltrust) [inline index.html:15]: Your "Roboto" fonts are not showing up even if you are declaring them on your CSS"

image

Upon checking the HTML, I can't see the "Roboto" fonts imported along with it's proper weights (I can see the Inter though).

- [ ] Please kindly import the "Roboto" fonts to your HTML along with the required font-weights so they actually show up on the page.  Don't forget to import the font weights that you need.

That way, your fonts will actually render on the page 💪

File: index.html [hunk 2]
```diff
@@ -54,24 +54,58 @@
...
           <li>Contact</li>
           <li><i class="fa fa-envelope"></i></li>
         </ul>
       </nav>
+
+      <div class="bar-icon">
+        <i class="fa fa-bars"></i>
+      </div>
+
+      <!--mobile menu -->
```
Comment (Meltrust) [inline index.html:75]: - [ ] Let's bring the margins/padding closer to the Figma template:

Your Implementation | Figma Design
:---:|:---:
image| image

As you can see, we need to add some margins/padding to bring the menu items and the close button a little lower, so the design ends up matching the Figma.

Also here:

- [ ] The arrows at the end of each menu item please kindly make them semi-transparent, like in the Figma.
- [ ] Make the close button a little bigger, so it matches the Figma template.

Look:

Your Implementation | Figma Design
:---:|:---:
image | image


That way, your mobile menu will look striking!! 💪 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Amedzro-Elikplim, 

## Your implementation is beautiful and almost on point, just follow my small comments below and this will be approved really fast!!

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mateo951) [reply index.html:15]: Addressed 🔥

File: index.html (hunk 2 above)
Comment (mateo951) [reply index.html:75]: Addressed ✔

**Review by mateo951 — APPROVED** [review]
### Hi Team,

Nice job implementing all the changes required by your previous reviewer 🤓
Your project is complete!  There is nothing else to say other than... it's time to merge it 🙌
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**